treeherder/etl/perf_data_adapters.py

```diff
@@ -223,23 +164,39 @@ def _transform_signature_properties(properties, significant_keys=None):
 
         return transformed_properties
 
-    def adapt_and_load(self, reference_data, job_data, datum):
+    def adapt_and_load(self, project_name, reference_data, job_data, datum):
+        if 'e10s' in reference_data.get('job_group_symbol', ''):
+            extra_properties = {'test_options': ['e10s']}
+        else:
+            extra_properties = {}
+
         # transform the reference data so it only contains what we actually
-        # care about
+        # care about (for calculating the signature hash reproducibly), then
+        # get the associated models
         reference_data = self._transform_signature_properties(reference_data)
+        option_collection = OptionCollection.objects.get(
+            option_collection_hash=reference_data['option_collection_hash'])
+        framework, _ = PerformanceFramework.objects.get_or_create(
```

Comment:
Should this be a straight get? A framework called talos should always exist, right?